src/Pmi/Controller/BiobankController.php

```diff
@@ -33,15 +33,28 @@ public function homeAction(Application $app)
 
     public function participantsAction(Application $app, Request $request)
     {
+        $bioBankIdPrefix = $app->getConfig('biobank_id_prefix');
         $idForm = $app['form.factory']->createNamedBuilder('id', Type\FormType::class)
             ->add('biobankId', Type\TextType::class, [
                 'label' => 'Biobank ID',
                 'constraints' => [
                     new Constraints\NotBlank(),
-                    new Constraints\Type('string')
+                    new Constraints\Type('string'),
+                    new Constraints\Callback(function($bioBankId, $context) use ($bioBankIdPrefix) {
+                        if (empty($bioBankIdPrefix)) {
+                            return;
+                        }
+                        $bioBankId = trim($bioBankId);
+                        $bioBankIdPrefix = preg_quote($bioBankIdPrefix, '/');
+                        if (!preg_match("/^{$bioBankIdPrefix}\d*$/", $bioBankId)) {
```

Comment:
Trivial, but wouldn't it `\d+` instead of `\d*` given that not providing a digit is invalid.